.pending/bugfixes/simulation/_Fix-non-determinism

```diff
@@ -0,0 +1 @@
+Fix non-determinism simulation by using CLI flags as input and updating Makefile target
```

Comment:
```suggestion
#4861 Fix non-determinism simulation by using CLI flags as input and updating Makefile target
```

Reply:
This will link it to an issue, not the PR. `clog` doesn't yet support this.